multiqc/templates/default/assets/js/plotting.js

```diff
@@ -143,12 +143,12 @@ function initPlot(dump) {
 }
 
 // Execute when page load has finished loading
-$(function () {
```

Comment:
This is the part that does not work properly. The `await` below is needed, but using the `async` keyword here does not magically make everything work.